test/instagram.realm.test.ts

```
@@ -0,0 +1,66 @@
+    undefined,
+    harness
+  );
+  expect(res.status).toBe(302);
+  expect(res.headers.get('location')).toBe('https://www.instagram.com/reel/CexampleReel/');
+});
+
+test('Instagram realm serves oembed', async () => {
+  const res = await app.request(
+    new Request(
+      'https://67instagram.com/owoembed?text=hello&status=CexampleShort&author=demo',
+      {
```

ESLint

[error] Replace `⏎······'https://67instagram.com/owoembed?text=hello&status=CexampleShort&author=demo',⏎·····` with `'https://67instagram.com/owoembed?text=hello&status=CexampleShort&author=demo',`

(prettier/prettier)